eureka2-client/src/main/java/com/netflix/eureka2/client/interest/BatchFunctions.java

```diff
@@ -22,10 +22,13 @@ private BatchFunctions() {
         return Observable.combineLatest(registryBatching, channelBatching, new Func2<BufferState, BufferState, Boolean>() {
```

Comment:
Can we express the state machine for this function in the comments above? This will help clarify the exact combine algorithm.